resources/event-sources/templates/uninstall.yaml

```diff
@@ -0,0 +1,110 @@
+# Copyright 2019 The Kyma Authors.
```

Comment:
General comment: It'd cool if we add labels in all the resources.

Reply:
Done